_release/sync-rtc.cron

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-exec /usr/bin/sync-rtc
+exec /usr/bin/rtc-utils --write-if-sync
```

Comment:
We also need something that runs straight after NTP time sync is achieved. There's a separate card for that already.